tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -55,3 +55,78 @@ def test_ndarray(self, xp, sp):
         b = xp.array(self.b, dtype=self.dtype)
         x = sp.linalg.lsqr(A, b)
         return x[0]
+
+
+@testing.parameterize(*testing.product({
+    'ord': [None, -numpy.Inf, -2, -1, 1, 2, 3, numpy.Inf, 'fro'],
+    'dtype': [
+        numpy.float32,
+        numpy.float64,
+        numpy.complex64,
+        numpy.complex128
+    ],
+}))
+@unittest.skipUnless(scipy_available, 'requires scipy')
+@testing.gpu
+class TestMatrixNorm(unittest.TestCase):
+
+    @testing.numpy_cupy_allclose(atol=1e-1, sp_name='sp', type_check=False,
+                                 accept_error=True)
+    def test_matrix_norm(self, xp, sp):
+        a = xp.arange(9, dtype=self.dtype) - 4
+        b = a.reshape((3, 3))
+        b = self._sparse_types(b, dtype=self.dtype)
+        return xp.array(sp.linalg.norm(b, ord=self.ord))
+
+    @testing.numpy_cupy_allclose(atol=1e-1, sp_name='sp', type_check=False,
+                                 accept_error=True)
+    def test_matrix_norm_axis_1(self, xp, sp):
+        a = xp.arange(9, dtype=self.dtype) - 4
+        b = a.reshape((3, 3))
+        b = sp.csr_matrix(b, dtype=self.dtype)
+        return xp.array(sp.linalg.norm(b, ord=self.ord, axis=None))
+
+    @testing.numpy_cupy_allclose(atol=1e-1, sp_name='sp', type_check=False,
+                                 accept_error=True)
+    def test_matrix_norm_axis_2(self, xp, sp):
+        a = xp.arange(9, dtype=self.dtype) - 4
+        b = a.reshape((3, 3))
+        b = sp.csr_matrix(b, dtype=self.dtype)
+        return xp.array(sp.linalg.norm(b, ord=self.ord, axis=(0, 1)))
+
+    @testing.numpy_cupy_allclose(atol=1e-1, sp_name='sp', type_check=False,
+                                 accept_error=True)
+    def test_matrix_norm_axis_3(self, xp, sp):
+        a = xp.arange(9, dtype=self.dtype) - 4
+        b = a.reshape((3, 3))
+        b = sp.csr_matrix(b, dtype=self.dtype)
+        return xp.array(sp.linalg.norm(b.T, ord=self.ord, axis=(1, 0)))
+
+
+@testing.parameterize(*testing.product({
+    'ord': [None, -numpy.Inf, -2, -1, 1, 2, numpy.Inf, 'fro'],
+    'dtype': [
+        numpy.float32,
+        numpy.float64,
+        numpy.complex64,
+        numpy.complex128
+    ],
+    'Transpose': [True, False],
```

Comment:
Could you use lowercase (`transpose`) for consistency?

Reply:
okay